config/Makefile

```diff
@@ -1,16 +1,24 @@
+# this is a region used for materializing configuration
```

Comment:
n.b. if we want to add another CI job that iterates through every possible value of (cloud,env,region) and validates the output for them, I think we could do that - the Go tool I wrote to do this for classic runs in milliseconds. We don't have to check in the output anywhere.